docs/ppp/iteration_3_01112022.md

```diff
@@ -0,0 +1,21 @@
+### Plans 📆
+
+- Build a Lightning Full node using LDK and Flutter and open a channel automatically [https://github.com/itchysats/10101/issues/10](https://github.com/itchysats/10101/issues/10).
+- Build out frontend based on the [wireframes](https://www.figma.com/file/zUBJpQ0v8cx6DHivS10OeI/TenTenOneWireFrame).
+- Add a custom output to a Lightning Channel. This will be the base for adding DLCs to a channel. [https://github.com/itchysats/10101/issues/54](https://github.com/itchysats/10101/issues/54)
```

Comment:
@luckysori are we going to publish the blog post? https://github.com/itchysats/10101/issues/14

Reply:
As discussed during our demo call, let's publish it during the next iteration 😉

Reply:
We can include it in this list then.